authorization/storage_authorization_test.go

```diff
@@ -77,6 +78,100 @@ func TestAuth(t *testing.T) {
+				for i := 1; i <= 10; i++ {
+					expectedAuth := &influxdb.Authorization{
+						ID:     influxdb.ID(i),
+						Token:  fmt.Sprintf("randomtoken%d", i),
+						OrgID:  influxdb.ID(i),
+						UserID: influxdb.ID(i),
+						Status: influxdb.Active,
+					}
+
+					authByID, err := store.GetAuthorizationByID(context.Background(), tx, influxdb.ID(i))
+					if err != nil {
+						t.Fatalf("Unexpectedly could not acquire Authorization by ID [Error]: %v", err)
+					}
+
+					if !reflect.DeepEqual(authByID, expectedAuth) {
+						t.Fatalf("ID TEST: expected identical authorizations:\n[Expected]: %+#v\n[Got]: %+#v", expectedAuth, authByID)
+					}
+
+					authByToken, err := store.GetAuthorizationByToken(context.Background(), tx, fmt.Sprintf("randomtoken%d", i))
+					if err != nil {
+						t.Fatalf("cannot get authorization by Token [Error]: %v", err)
+					}
+
+					if !reflect.DeepEqual(authByToken, expectedAuth) {
+						t.Fatalf("TOKEN TEST: expected identical authorizations:\n[Expected]: %+#v\n[Got]: %+#v", expectedAuth, authByToken)
+					}
+				}
+
+			},
+		},
+		{
+			name:  "update",
+			setup: setup,
+			update: func(t *testing.T, store *authorization.Store, tx kv.Tx) {
+				for i := 1; i <= 10; i++ {
+					auth, err := store.GetAuthorizationByID(context.Background(), tx, influxdb.ID(i))
+					if err != nil {
+						t.Fatalf("Could not get authorization [Error]: %v", err)
+					}
+
+					auth.Status = influxdb.Inactive
+
+					_, err = store.UpdateAuthorization(context.Background(), tx, influxdb.ID(i), auth)
+					if err != nil {
+						t.Fatalf("Could not get updated authorization [Error]: %v", err)
+					}
+				}
+			},
+			results: func(t *testing.T, store *authorization.Store, tx kv.Tx) {
+
+				for i := 1; i <= 10; i++ {
+					auth, err := store.GetAuthorizationByID(context.Background(), tx, influxdb.ID(i))
+					if err != nil {
+						t.Fatalf("Could not get authorization [Error]: %v", err)
+					}
+
+					expectedAuth := &influxdb.Authorization{
+						ID:     influxdb.ID(i),
+						Token:  fmt.Sprintf("randomtoken%d", i),
+						OrgID:  influxdb.ID(i),
+						UserID: influxdb.ID(i),
+						Status: influxdb.Inactive,
+					}
+
+					if !reflect.DeepEqual(auth, expectedAuth) {
+						t.Fatalf("expected identical authorizations:\n[Expected] %+#v\n[Got] %+#v", expectedAuth, auth)
+					}
+				}
+			},
+		},
+		{
+			name:  "delete",
+			setup: setup,
+			update: func(t *testing.T, store *authorization.Store, tx kv.Tx) {
+				for i := 1; i <= 10; i++ {
+					err := store.DeleteAuthorization(context.Background(), tx, influxdb.ID(i))
+					if err != nil {
+						t.Fatalf("Could not delete authorization [Error]: %v", err)
+					}
+				}
+			},
+			results: func(t *testing.T, store *authorization.Store, tx kv.Tx) {
+				for i := 1; i <= 10; i++ {
+					_, err := store.GetAuthorizationByID(context.Background(), tx, influxdb.ID(i))
+					if err == nil {
+						t.Fatalf("Authorization was not deleted correctly [Error]: %v", err)
```

Comment:
since this print statement is in an `err == nil` check, it will always just print nil here. you can just leave off the %v err part.

Reply:
O yeah that makes sense. I'm on it!